data/wanderer/wanderers start.txt

```diff
@@ -2298,6 +2298,40 @@ mission "Wanderers Evacuation 1B"
 
 
 
+mission "Wanderers Evacuation 1B Remove Facilities"
+	landing
+	invisible
+	source "Vara Ke'stai"
+	to offer
+		has "Wanderers Evacuation 1: done"
+	on offer
+		event "remove vara ke'stai facilities"
+
+event "remove vara ke'stai facilities"
+	planet "Vara Ke'stai"
+		remove spaceport
+		remove outfitter
+		remove "required reputation"
+		remove tribute
+		add port
+			recharges fuel
+			description `This is a desert world, but the Wanderers are working little by little to make the desert bloom: planting hardy shrubs and cacti that eventually transform the sand into a soil that can hold moisture. In some regions, especially closer to the poles, the deserts have given way to dense forests, but near the equator there are still large stretches of dry and barren sand, broken by the occasional belt of green where a narrow river winds between the dunes.`
```

Comment:
The description in here is shown on the spaceport panel, not the normal landing description.